README.md

```diff
@@ -55,20 +56,61 @@ ionice -c 3 nice -n 19 log4j-sniffer crawl /path/to/a/directory
 Output for vulnerable files looks as follows:
 
 ```
-CVE-2021-45046 and CVE-2021-45105 detected in file examples/single_bad_version/log4j-core-2.14.1.jar. log4j versions: 2.14.0 - 2.14.1, 2.14.1. Reasons: jar name matched, class and package name matched, class file MD5 matched
+[INFO] Found archive with name matching vulnerable log4j-core format at examples/single_bad_version/log4j-core-2.14.1.jar
+[INFO] Found JndiManager class that was an exact md5 match for a known version at org/apache/logging/log4j/core/net/JndiManager.class
+[INFO] Found JndiLookup class in the log4j package at org/apache/logging/log4j/core/lookup/JndiLookup.class
+[MATCH] CVE-2021-45046 and CVE-2021-45105 detected in file examples/single_bad_version/log4j-core-2.14.1.jar. log4j versions: 2.14.0-2.14.1, 2.14.1. Reasons: JndiLookup class and package name matched, jar name matched, JndiManager class and package name matched, class file MD5 matched
 Files affected by CVE-2021-45046 or CVE-2021-45105 detected: 1 file(s) impacted by CVE-2021-45046 or CVE-2021-45105
 1 total files scanned, skipped 0 paths due to permission denied errors, encountered 0 errors processing paths
 ```
 
-CVE-2021-45105
-==============
+Getting started
+===============
 
-If you do not wish to report results for CVE-2021-45105 then pass the `--disable-cve-2021-45105-detection` flag to the crawl command.
+### Mac
 
-By default, both CVE-2021-45046 and CVE-2021-45105 will be reported.
+##### Download the latest version
+
+1. Locate log4j-sniffer/releases (https://github.com/palantir/log4j-sniffer/releases).
+2. You will need a different asset depending on the generation of your Mac.
+    1. Select the asset with “macos-amd” in the file name for older Intel Macs.
+    2. Select “macos-arm” for newer m1 Macs
+3. Confirm that the file is downloading to your “Downloads” folder.
+
+##### Install log4j-sniffer on your machine
+
+1. Once the download is complete, click on the file to open.
+1. Drag and drop the “log4j-sniffer” icon into your Downloads through the Finder.
+    1. Open a Finder window by searching for “Finder” using the magnifying glass on the top right of your screen, or selecting the icon in your Dock.
+    2. Drag and drop the “log4j-sniffer” icon into Downloads.
+ ![dragging and dropping](https://github.com/palantir/log4j-sniffer/blob/develop/mac_download.png?raw=true)
+
+##### Use log4j-sniffer
+
+1. Open the Terminal by searching for “Terminal” using the magnifying glass in the top right corner of the screen.
+2. Run `~/Downloads/log4j-sniffer crawl ~/ --ignore-dir="^/dev"` to crawl the entire system
```

Comment:
```suggestion
2. Run `~/Downloads/log4j-sniffer crawl / --ignore-dir="^/dev"` to crawl the entire system
```

Reply:
Adding ~ will ensure it only scans the Users folder, think we want to recommend scanning the whole system?